schemas/dc-library.xsd

```diff
@@ -94,6 +97,7 @@
       <xs:simpleContent>
         <xs:extension base="xs:string">
           <xs:attribute name="root" type="xs:string" use="optional"/>
+          <xs:attribute name="technical" type="xs:string" use="optional"/>
```

Comment:
I made the change in `meta`, my understanding is that `cite` still needs it. Is this correct?